doc/source/release-notes/Percona-Server-5.7.28-31.rst

```diff
@@ -0,0 +1,59 @@
+.. rn:: 5.7.28-31:
+
+===============================================================================
+|Percona Server| |release|
+===============================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on |date|. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.28-31/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.27 <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-28.html>`_ and includes all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+
+All software developed by Percona is open-source and free.
+
+.. note::
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+Bugs Fixed
+===============================================================================
+
+- When using `skip-innodb_doublewrite` in my.cnf, a parallel doublewrite buffer is still created. Bugs fixed :psbug:`3411`.
+
+- During a binlogging replication event, if the master crashes after the multi-threaded slave has begun copying to the slave's relay log and before the process has completed, a `STOP SLAVE` on the slave takes longer than expected. Bug fixed :psbug:`5824`.
+
+- If pam_krb5 is configured to allow the user to change their password, and the password is expired, the server crashed after receiving the new password. Bug fixed :psbug:`6023`.
```

Comment:
what is the `pam_krb5`? user friendly explanation or link to more info would be useful.

Reply:
added link to https://docs.oracle.com/cd/E88353_01/html/E37853/pam-krb5-7.html